### ISSUE TITLE
How to get full graph?

### ISSUE THREAD

**Issue by fdiotalevi** [ltearno/pom-explorer#35]
I was expecting to get the full graph of dependencies (including all the transitive dependencies of all dependencies) with this code

        PomFileLoader pomFileLoader = new DefaultPomFileLoader(session.session(), true);
        PomAnalysis.runFullRecursiveAnalysis(args[0], session.session(),
                pomFileLoader,
                new String[0],
                true, log);

but realised that gives me only the declared dependencies. Is there a way to achieve that?

**Comment by ltearno** [comment]
Yes, this ones only loads the pom files, without running the transitive dep analysis (because for this you sometimes need to load additionnal projects - well the transitive dependencies).

You have a unit test that does that (the `AnalyzerTest:test05` method) here : https://github.com/ltearno/pom-explorer/blob/master/pom-explorer-core/src/test/java/fr/lteconsulting/pomexplorer/AnalyzerTest.java#L73.

Basically it boils down to adding :

	TransitivityResolver resolver = new TransitivityResolver();
	resolver.getTransitiveDependencyTree( session, project, true, true, null, new PomFileLoader()
	{
		@Override
		public File loadPomFileForGav( Gav gav, List<Repository> additionalRepos, Log log )
		{
			...
		}
	}, System.out::println );
